shopinvader/views/sale_view.xml

```diff
@@ -11,9 +11,9 @@
                 <field name="current_step_id" attrs="{'invisible': [('typology', '!=', 'cart')]}"/>
                 <field name="done_step_ids" widget="many2many_tags" attrs="{'invisible': [('typology', '!=', 'cart')]}"/>
             </field>
-            <!-- field name="company_id" position="before">
-                <field name="shopinvader_backend_id"/>
-            </field -->
+            <xpath expr="//page[@name='other_information']//group[@name='sales_person']" position="inside">
+                <field name="shopinvader_backend_id" groups="shopinvader.group_shopinvader_manager"/>
```

Comment:
Is the seecurity group really required. It could be a really interesting information for all users.
I would replace it by a readonly attribute.

Reply:
+1 
["sale.order"].backend_id should be flagged as readonly